Cargo.toml

```diff
@@ -10,6 +10,6 @@ readme = "Readme.md"
 keywords = ["image", "metadata"]
 
 [dependencies]
-byteorder = "0.4"
+byteorder = "0.5.*"
```

Comment:
Actually `0.5` would be `^0.5` or, identically, `^0.5.0`, which means any version of `0.5.x` kind. So just `0.5` includes all the patch versions.